libs/SalesforceSDK/build.gradle

```diff
@@ -18,6 +18,7 @@ dependencies {
     api 'com.squareup.okhttp3:okhttp:3.9.0'
     api 'com.google.android.gms:play-services-gcm:11.8.0'
     api 'com.android.support:customtabs:25.2.0'
+    androidTestApi 'junit:junit:4.11'
```

Comment:
<table data-meta="generated_by_SalesforceSDK">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/forcedotcom/SalesforceMobileSDK-Android/blob/7e76c91d63aa9cd6655127901e5ce32158ce9b54/libs/SalesforceSDK/build.gradle#L21"></span>A newer version of junit:junit than 4.11 is available: 4.12</td>
    </tr>
  </tbody>
</table>